src/krkn_lib/telemetry/elastic.py

```diff
@@ -0,0 +1,37 @@
+import time
+from krkn_lib.utils.safe_logger import SafeLogger
+from elasticsearch import Elasticsearch
+
+class KrknElastic:
+    es = None
+
+    def __init__(
+        self,  safe_logger: SafeLogger, elastic_url: str
+    ):
+        self.safe_logger = safe_logger
+        try: 
+            # create Elasticsearch object
+            self.es = Elasticsearch(f'{elastic_url}:443')
+        except Exception as e: 
+            self.safe_logger.error("Failed to initalize elasticsearch: %s" % e)
+            raise e
+
+    def upload_data_to_elasticsearch(self,item, index=''):
```

Comment:
Could you please add parameters types and pydoc :param: and :return: descriptions?

Reply:
also some linting (space missing after comma) sorry :-)